tachyon/math/elliptic_curves/short_weierstrass/point_xyzz.h

```diff
@@ -109,6 +111,35 @@ class PointXYZZ<_Curve,
                      point.y_, point.zz_, point.zzz_);
   }
 
+  // TODO(chokobole): Implement parallel versioned |BatchNormalize| when doing
+  // chunking and zipping gets easier.
+  template <typename PointXYZZContainer, typename AffineContainer>
+  [[nodiscard]] constexpr static bool BatchNormalize(
+      const PointXYZZContainer& point_xyzzs, AffineContainer* affine_points) {
+    size_t size = std::size(point_xyzzs);
+    if (size != std::size(*affine_points)) {
+      LOG(ERROR) << "Size of |point_xyzzs| and |affine_points| do not match";
+      return false;
+    }
+    std::vector<BaseField> zzz_inverses = base::Map(
+        point_xyzzs, [](const PointXYZZ& point) { return point.zzz_; });
+    if (!BaseField::BatchInverseInPlaceSerial(zzz_inverses)) return false;
+    for (size_t i = 0; i < size; ++i) {
+      const BaseField& z_inv_cubic = zzz_inverses[i];
```

Comment:
```suggestion
      const BaseField& zzz_inv = zzz_inverses[i];
```
or
```suggestion
      const BaseField& z_cubic_inv = z_cubic_inverses[i];
```

Reply:
Aren't `z_inv_cubic` and `z_cubic_inv` same? Plus, this name is same with the ones used in `ToAffine()`.